### PR TITLE
FIRST AC done -- now ok

### PR DESCRIPTION
**Pull request by nannatalia**
pls review

### CODE REVIEW

**Review by rokudaimme — APPROVED** [review]

**Comment by rokudaimme** [comment]
good!

**Review by Jepbarova94 — APPROVED** [review]

File: src/test/resources/features/search.feature [hunk 1]
```diff
@@ -4,8 +4,6 @@ Feature: As a user, I should be able to search file
   
 Background: Given user is on Dashboard page  
 
-Background: Given user is on Dashboard page
```
Comment (Jepbarova94) [inline src/test/resources/features/search.feature]: I am not sure but it seems that you have 2 Background

**Review by Jepbarova94 — COMMENTED** [review]
There is 2 Background in the feature file on line 5 and 7

File: src/test/resources/features/search.feature (hunk 1 above)
Comment (nannatalia) [reply src/test/resources/features/search.feature]: if its in red it is deleted

**Pull request merged**